src/test/java/org/broadinstitute/hellbender/tools/picard/fasta/dek_Normalizefasta_IT.java

```diff
@@ -0,0 +1,41 @@
+package org.broadinstitute.hellbender.tools.picard.fasta;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.tools.IntegrationTestSpec;
+import org.broadinstitute.hellbender.tools.picard.fastq.NormalizeFasta;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.annotations.Test;
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * Created by dkling on 9/2/15.
+ */
+
+    public final class dek_Normalizefasta_IT extends CommandLineProgramTest {
+
+        private static final File TEST_DATA_PATH = new File(getTestDataDir(), "picard/fasta/" );
+
+        public String getTestedClassName() {
+            return NormalizeFasta.class.getSimpleName();
+        }
+
+        @Test
+        public void testNormalize () throws IOException {
+
+            final File input = new File(TEST_DATA_PATH, "exampleFASTA.fasta");
+            final File expectedFile = new File(TEST_DATA_PATH, "Normalizedexp.fasta");
+            final File outfile = BaseTest.createTempFile("normalized", ".fasta");
+
+            final String[] args = {
+                    "--INPUT", input.getAbsolutePath(),
+                    "--OUTPUT", outfile.getAbsolutePath(),
+            };
+
+            runCommandLine(args);
+            IntegrationTestSpec.assertEqualTextFiles(outfile, expectedFile, "#");
+
+        }
+
```

Comment:
Please clarify what you mean by "not passing...".  I added (possibly back in) "(expectedExceptions = UserException.BadArgumentValue.class)" after @Test.  This enabled Hellbender to handle LIne_Length=null.  If I remove the code containing Line_Length=null, I also have to remove, " (expectedExceptions = UserException.BadArgumentValue.class)" after @Test, in order for the program to run.